department-of-veterans-affairs/va.gov-cms
# VACMS-8016 time widget & alert icon updates

## Description

Relates to #8016. Last remaining high/medium issues in the airtable :tada: 

- Adjusts spacing of inputs inside the Hours field to be easier to parse, more consistent with current experience
- Adjusts color of status alert message to match vagovadmin (green instead of light blue)
- Updates icons for alert messages... claro prints these as part of the message header, but i was asked to hide them because they were redundant. So the icons vanished. Now they're back!

## Testing done
- checked time widget spacing at all screen sizes. Behavior now matches vagovadmin at all sizes (stacks on lower, inline on higher screen sizes)
- verified icons appear for all types of alerts and that node link reports still have the toggle icon

## Screenshots
Time widget updates:
<img width="789" alt="Screen Shot 2022-02-16 at 10 02 09 AM" src="https://user-images.githubusercontent.com/11279744/154327404-aae0daad-b746-440a-88b5-7c2d0fd5eed4.png">
<img width="1025" alt="Screen Shot 2022-02-16 at 10 02 17 AM" src="https://user-images.githubusercontent.com/11279744/154327407-94bc776a-3250-4c1a-9a65-0aa91ef04b35.png">

Alert updates:
<img width="1127" alt="Screen Shot 2022-02-16 at 8 49 37 AM" src="https://user-images.githubusercontent.com/11279744/154327212-8cc8f8fa-f689-4167-8ae5-47ee851fdfbd.png">
<img width="1119" alt="Screen Shot 2022-02-16 at 8 55 57 AM" src="https://user-images.githubusercontent.com/11279744/154327214-a07cbb94-d693-49c2-a9c7-349123a222fb.png">
<img width="1130" alt="Screen Shot 2022-02-16 at 8 56 10 AM" src="https://user-images.githubusercontent.com/11279744/154327216-40e47fe0-c55a-417a-b26e-bb9fa4cfbecc.png">
<img width="817" alt="Screen Shot 2022-02-16 at 8 56 46 AM" src="https://user-images.githubusercontent.com/11279744/154327219-d9550f13-1dc6-442a-88d1-7e28944cb436.png">
<img width="451" alt="Screen Shot 2022-02-16 at 9 33 20 AM" src="https://user-images.githubusercontent.com/11279744/154327221-888c8bb9-90d6-409b-8536-4cb7320d72cc.png">


## QA steps
- log in as admin
- visit `node/add/health_care_local_facility` and scroll down to the "Hours" section
- see that the spacing is inline on higher screen sizes, layout should be easier to parse at a glance

- visit `node/43232/edit` and see node link report, content lock alert
- visit `admin/appearance` and see the warning alert
- visit `node/add/page` and add a table paragraph to the main content block. rebuild the table rows to see the inline alert
- log out and attempt to log back in with wrong credentials to see the error alert

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
